src/parser/jobs/blu/modules/MoonFlute.tsx

```diff
@@ -132,7 +138,27 @@ export class MoonFlute extends BuffWindow {
 				are <ActionLink action="NIGHTBLOOM" showIcon={false} />, and finishing the combo with a <ActionLink action="PHANTOM_FLURRY" showIcon={false} />.
 			</Trans>,
 			severityTiers: SEVERITIES.MISSING_EXPECTED_USES,
+			adjustOutcome: this.adjustExpectedActionOutcome.bind(this),
 		}))
 	}
+
+	private adjustExpectedGcdCount(window: HistoryEntry<EvaluatedAction[]>) {
+		const finalStingUsed = window.data.filter(event => (event.action.id === this.data.actions.FINAL_STING.id || event.action.id === this.data.actions.SELF_DESTRUCT.id)).length
+		return finalStingUsed >= 1 ? (-window.data.length+1) : 0
+	}
+
+	private adjustExpectedActionOutcome(window: HistoryEntry<EvaluatedAction[]>, _action: TrackedActionGroup) {
+		const finalStingUsed = window.data.filter(event => (event.action.id === this.data.actions.FINAL_STING.id || event.action.id === this.data.actions.SELF_DESTRUCT.id)).length
```

Comment:
Minor nit: Since you're running the same logic in both adjust functions, you might consider extracting that to a helper function

Reply:
Done!